### PR TITLE
Fix remaining ESLint errors in server/__tests__ directory

### PR DESCRIPTION
**Pull request by ajhyndman**
## Description

This pull request removes ignore rules for the last remaining JavaScript and Vue files in the packages/server/__tests__ directory of this repository.

Note that all changes in this pull request were manually applied. Everything that could be resolved via eslint --fix was committed in https://github.com/usdigitalresponse/usdr-gost/pull/1036.

## Screenshots / Demo Video

BEFORE

image

AFTER

image

## Testing

Test these changes by running yarn lint in the packages/server directory.

### Automated and Unit Tests
- ~Added Unit tests~

### Manual tests for Reviewer
- [x] Added steps to test feature/functionality manually

## Checklist
- [x] Provided ticket and description
- [x] Provided screenshots/demo
- [x] Provided testing information
- ~Provided adequate test coverage for all new code~
- [x] Added PR reviewers

### CODE REVIEW

File: packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js [hunk 1]
```diff
@@ -119,51 +120,49 @@ describe('validate record', () => {
...
+    it('validates a valid project succesfully', () => validateRecord({
+        upload: { ec_code: '2.16' },
+        record: VALID_EC2_PROJECT,
+        typeRules: ALL_RULES.ec2,
+    }).then(
+        (generatedErrors) => {
+            assert(generatedErrors.length === 0,
+                `Unexpected error when validating record: ${generatedErrors}`);
+        },
+        (thrownException) => { fail('Unexpected error while validating record', thrownException); },
```
Comment (ajhyndman) [inline packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js:132]: @trumanc Any chance you remember where this `fail` function should be coming from?

I noticed that `chai` and `assert` libraries both provide a `fail` method, but neither signature seems to match your usage here.

**Review by NatHillardUSDR — APPROVED** [review]
Nice! Thanks again for taking this on. I'll defer to Truman on the `fail` question

File: packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js (hunk 1 above)
Comment (trumanc) [reply packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js:132]: oh whoops, it looks like I meant for that to be a single string that included the exception info, not a second parameter that breaks the method signature. You could change this to

```(thrownException) => { fail(`Unexpected error while validating record: ${thrownException}`); },```

Comment (ajhyndman) [reply packages/server/__tests__/arpa_reporter/server/services/validate-upload.spec.js]: Awesome, thanks for clarifying!

**Pull request merged**